<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingWidget</class>
 <widget class="QWidget" name="settingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>314</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel{
font-family:&quot;微软雅黑&quot;;
font-size:12px;
color:#868989;
line-height:12px;
text-align:right;
}
QWidget#widget{
background:#eeeeee;
box-shadow:0 2px 10px 1px rgba(0,0,0,0.20);
border-bottom-left-radius:4px;
border-bottom-right-radius:4px;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>90</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>100</number>
      </property>
      <property name="bottomMargin">
       <number>40</number>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <property name="horizontalSpacing">
         <number>20</number>
        </property>
        <property name="verticalSpacing">
         <number>14</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>48</width>
            <height>12</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>48</width>
            <height>12</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>设备别名</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="3">
         <widget class="commonLineEdit" name="deviceName" native="true">
          <property name="minimumSize">
           <size>
            <width>220</width>
            <height>29</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>48</width>
            <height>12</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>48</width>
            <height>12</height>
           </size>
          </property>
          <property name="text">
           <string>预置画面</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" rowspan="2" colspan="3">
         <widget class="QWidget" name="imageWidget" native="true">
          <property name="minimumSize">
           <size>
            <width>220</width>
            <height>110</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_3">
          <property name="minimumSize">
           <size>
            <width>48</width>
            <height>12</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>48</width>
            <height>12</height>
           </size>
          </property>
          <property name="text">
           <string>自动旋转</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="autoBtn">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>14</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>14</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image:url(:/image/image/effects.png);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="cancelBtn">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>26</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">#cancelBtn{
background:#ffffff;
border-radius:2px;
font-family:&quot;微软雅黑&quot;;
font-size:12px;
color:#323434;
letter-spacing:0;
line-height:12px;
text-align:center;
}
#cancelBtn:pressed{
margin:1px;
}
</string>
          </property>
          <property name="text">
           <string>取消</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="confirmBtn">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>26</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">#confirmBtn{
background:#ffd500;
border-radius:2px;
width:80px;
height:26px;
font-family:&quot;微软雅黑&quot;;
font-size:12px;
color:#323434;
letter-spacing:0;
line-height:12px;
text-align:center;
}
#confirmBtn:pressed{
margin:1px;
}</string>
          </property>
          <property name="text">
           <string>确定</string>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>commonLineEdit</class>
   <extends>QWidget</extends>
   <header>commonlineedit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
